charts/sbombastic/questions.yaml

```diff
@@ -75,6 +75,22 @@ questions:
     description: |
       Log level of the Worker Deployment
 
+  - variable: worker.trivyDBRepository
+    label: Worker Trivy DB Repository
+    type: string
+    default: public.ecr.aws/aquasecurity/trivy-db
+    group: Worker
+    description: |
+      OCI repository to retrieve trivy-db
```

Comment:
Are we sure this is an OCI repository? AFAIK it's also possible to pull this data from an S3 bucket